pandamium_datapack/data/pandamium/functions/tpa/accept_request.mcfunction

```diff
@@ -1,4 +1,8 @@
-execute as @p[tag=running_trigger] run function pandamium:tpa/check_can_accept
+tag @s add sender
+tag @p[tag=running_trigger] add receiver
+function pandamium:tpa/check_can_accept
+tag @a remove sender
```

Comment:
why not use @s here? does the context change since line 1?
that would be cleaner imo, and maybe slightly more performance

same with @p one line below

Reply:
ah I just saw it's also used below, in that case just using @a everywhere is probably the easiest